### DISCUSSION TITLE
How can i change imprint and data-privacy-policy?

### DISCUSSION THREAD

**Discussion by legrex** [OpenSlides/OpenSlides#6678] [Q&A] [answered]
Hi all,

i guess im blind or i did miss something, but how can i change the imprint- and data-privacy-policy-data? I'm running 4.* (installed 3 days ago), but i don't see where to look for this. :/

Thank you!

**Comment by jsangmeister** [comment, marked as answer]
I'm assuming you mean the "Legal notice" and "Privacy Policy" undeer, e.g., https://demo.os4.openslides.com/info? You have to be an organization admin, then there is a pen icon on the top right corner of both sections to edit them. I hope this helps you.

**Reply by legrex** [reply]
Thank you, i indeed missed to click this link as i was logged in as admin, i just searched in settings and so. Of course my fault, thanks for your reply!
